setup.py

```diff
@@ -1,10 +1,12 @@
 from setuptools import setup, find_packages
+from io import open
```

Comment:
I'd rather we didn't overwrite the `open` built-in, because I just find that confusing.